tests/e2e/cucumber/features/smoke/app-provider/officeSuites.ocis.feature

```diff
@@ -0,0 +1,35 @@
+Feature: Integrate with online office suites like Collabora and OnlyOffice
+  As a user
+  I want to work on different docs, sheets, slides etc.., using online office suites like Collabora or OnlyOffice
+  So that the collaboration is seamless
+
+  # To run this feature we need to run the external app-provider service along with wopi, OnlyOffice, Collabora services
+  # This is a minimal test for the integration of ocis with different online office suites like Collabora and OnlyOffice
+
+  Background:
+    Given "Admin" creates following users using API
+      | id    |
+      | Alice |
+    And "Alice" logs in
+    And "Alice" opens the "files" app
+    And "Alice" navigates to the personal space page
+
+  Scenario: create an OpenDocument file with Collabora
+    When "Alice" creates the following resources
+      | resource         | type         | content              |
+      | OpenDocument.odt | OpenDocument | OpenDocument Content |
+    And "Alice" creates a public link for the resource "OpenDocument.odt" using the sidebar panel
+    And "Alice" edits the public link named "Link" of resource "OpenDocument.odt" changing role to "Can edit"
+    And "Alice" logs out
+    And "Anonymous" opens the public link "Link"
+    Then for "Anonymous" the content of the file "OpenDocument.odt" should be "OpenDocument Content" in editor "Collabora"
+
+  Scenario: create a Microsoft Word file with OnlyOffice
```

Comment:
```suggestion
  Scenario: create a Microsoft Word file with OnlyOffice
```
do we need to use capital letters in `Microsoft Word` and `OnlyOffice`

Reply:
to make it more generic